reference/shipping_provider.yml

```diff
@@ -1984,14 +1984,12 @@ components:
                     type: string
             customer_id:
               type: integer
-        cart_id:
-          type: string
         store_id:
           type: string
         request_context:
           type: object
           title: Request Context
-          description: A collection of Reference Value objects.
+          description: A collection of Reference Value objects. Two examples of this is the cart_id and the channel_id.
```

Comment:
@gazzayeatman One question I had was whether this would be a valid example for `request_context`: 

`"request_context": {
   "reference_values": [
      {
         "name": "channel_id",
          "value": "1"
      },
      {
         "name": "cart_id",
          "value": "abc"
      },
   ]
}     
`

Reply:
Correct, it would. The default channel ID is also 1